<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThinLensCameraWidget</class>
 <widget class="QWidget" name="ThinLensCameraWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="angle_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Angle of view</string>
   </property>
   <widget class="QDoubleSpinBox" name="angle_spinbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string>Ëš</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>179.000000000000000</double>
    </property>
    <property name="value">
     <double>45.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="aperture_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Aperture</string>
   </property>
   <widget class="QDoubleSpinBox" name="aperture_spinbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string>m</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>0.000001000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.000001000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="focal_distance_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>91</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Focal distance</string>
   </property>
   <widget class="QDoubleSpinBox" name="focal_distance_spinbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="buttonSymbols">
     <enum>QAbstractSpinBox::NoButtons</enum>
    </property>
    <property name="keyboardTracking">
     <bool>false</bool>
    </property>
    <property name="suffix">
     <string>m</string>
    </property>
    <property name="decimals">
     <number>6</number>
    </property>
    <property name="minimum">
     <double>0.000001000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
    <property name="value">
     <double>0.050000000000000</double>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
